kex-core/src/main/kotlin/org/vorpal/research/kex/trace/symbolic/SymbolicTraceBuilder.kt

```diff
@@ -423,6 +424,8 @@ class SymbolicTraceBuilder(
     }
 
     override fun track(value: String, concreteValue: Any?) {
+        // clear cache - some earlier versions of values may be loaded
```

Comment:
1. Instead of accessing internal implementation of converter each time it is easier to just create new converter
2. Resetting converter cache before each `track` is not quite correct, because we may lose some important connections between variables. In general it would be more correct to add external way to reset cache whenever we need it. But I would say that currently it is more correct to reset converter each time we see a return instruction